modules/installation-supported-aws-machine-types.adoc

```diff
@@ -466,3 +482,8 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 
 |===
 ====
+endif::[]
+
+ifeval::["{context}" == "installing-restricted-networks-aws.adoc"]
+:!arm-upi:
```

Comment:
Where is this conditional defined or used?

Reply:
That was my mistake, I meant to change that to the conditional I used. Will fix